formats/sarifutils/sarifutils.go

```diff
@@ -19,12 +17,30 @@ func NewReport() (*sarif.Report, error) {
 	return report, nil
 }
 
-func ConvertSarifReportToString(report *sarif.Report) (sarifStr string, err error) {
-	out, err := json.Marshal(report)
-	if err != nil {
-		return "", errorutils.CheckError(err)
+func NewPhysicalLocation(physicalPath string) *sarif.PhysicalLocation {
+	return &sarif.PhysicalLocation{
+		ArtifactLocation: &sarif.ArtifactLocation{
+			URI: &physicalPath,
+		},
+	}
+}
+
+func NewPhysicalLocationWithRegion(physicalPath string, startRow, endRow, startCol, endCol int) *sarif.PhysicalLocation {
```

Comment:
why do we pass the integers by ref and not value? its into an object you create now - will it change afterwards?

Reply:
yes, this is required by the library we use, it holds pointers